static/css/custom.css

```diff
@@ -706,6 +706,10 @@ s  }
   margin: -1px;
 }
 
+.quire-page__content .container .content .source {
```

Comment:
I'd recommend adding a brief comment here so future contributors know what this small bit of CSS is targeting.

And let's make the top margin 6rem, the same as H2 margins, which separate the sections within the page.

Reply:
Ok it seems logic!